examples/reminders/lib/base/repositories/reminders_repository.dart

```diff
@@ -24,16 +24,18 @@ class RemindersRepository {
     required String title,
     required DateTime dueDate,
     required bool complete,
+    required bool completeUpdated,
```

Comment:
As we discussed no extra props should be added

Reply:
This property has been removed